packages/ui/src/views/repo/repo-tags/components/dialogs/create-tag/hooks/types.ts

```diff
@@ -0,0 +1,5 @@
+export type UseCreateTagFormProps = {
```

Comment:
This type is only used in `use-create-tag-form.ts`. Maybe it is better to keep things close to each other
Are you going to extend this file with other types?